stable/admin/README.md

```diff
@@ -182,10 +182,13 @@ The following tables list the configurable parameters for the `admin` option of
 | `tolerations` | Tolerations for NuoDB Admin | `[]` |
 | `configFilesPath` | Directory path where `configFiles.*` are found | `/etc/nuodb/` |
 | `configFiles.*` | See below. | `{}` |
-| `persistence.enabled` | Whether or not persistent storage is enabled for admin RAFT state | `false` |
 | `persistence.accessModes` | Volume access modes enabled (must match capabilities of the storage class) | `ReadWriteMany` |
 | `persistence.size` | Amount of disk space allocated for admin RAFT state | `10Gi` |
 | `persistence.storageClass` | Storage class for volume backing admin RAFT state | `-` |
+| `logPersistence.enabled` | Whether to enable persistent storage for logs | `20Gi` |
```

Comment:
`enabled` sounds like a boolean option. But this takes a file size string. How do you disable this?

Reply:
copy n paste error, will fix